Project/Source/Presentation/Scene/Main/Cell/MainTableViewCell.swift

```diff
@@ -0,0 +1,99 @@
+import UIKit
+import Then
+import SnapKit
+
+protocol hearBtnTableVeiwCellDelegate {
+    func heartBtnDidTap(id: String) -> Bool
+}
+
+final class MainTableViewCell: baseTableViewCell<MainModel> {
+    
+    var delegate: hearBtnTableVeiwCellDelegate?
+    
+    private let postImageView = UIImageView().then {
+        $0.contentMode = .scaleAspectFit
+        $0.image = UIImage(systemName: "doc.badge.plus")
+        $0.backgroundColor = .init(red: 0.92, green: 0.92, blue: 0.92, alpha: 1)
+        $0.layer.cornerRadius = 12
+        $0.layer.maskedCorners = CACornerMask(arrayLiteral: .layerMinXMinYCorner, .layerMaxXMinYCorner)
```

Comment:
```suggestion
        $0.layer.maskedCorners = [.layerMinXMinYCorner, .layerMaxXMinYCorner]
```
로 줄이는게 더 가독성이 좋을거 같은데 어떤가요?

Reply:
그렇게도 바꿀 수 있네요. 감사합니다

Reply:
[d6acce0](https://github.com/gabozzago/gabozzago-ios/pull/5/commits/d6acce06ef866aa0045eb342550f6150cf0c438f)